models/demos/tg/llama3_70b/tests/test_llama_decoder_galaxy.py

```diff
@@ -114,8 +114,8 @@ def tt_llama_decoder_prepare_inputs(llama_decoder_model, x, start_pos, mode):
         x = x.transpose(0, 1).unsqueeze(1)  # [seq_len, 1, batch, hidden_dim]
 
         ACT_MEMCFG = ttnn.create_sharded_memory_config(
-            shape=(x.shape[2], x.shape[3] // 8 // llama_decoder_model.cluster_shape[0]),
-            core_grid=ttnn.CoreGrid(y=1, x=8),
+            shape=(x.shape[2], x.shape[3] // 32 // llama_decoder_model.cluster_shape[0]),
```

Comment:
Can we put num_devices instead of 32. Same applies for models/demos/tg/llama3_70b/tt/llama_model_galaxy.py.

Reply:
Otherwise looks good to me!

Reply:
32 is num_cores not devices, I can parameterize it